sql/mob_droplist.sql

```diff
@@ -16666,6 +16664,24 @@ INSERT INTO `mob_droplist` VALUES (3149,0,0,1000,15145,10);
 INSERT INTO `mob_droplist` VALUES (3149,0,0,1000,15146,10);
 INSERT INTO `mob_droplist` VALUES (3149,0,0,1000,16349,10);
 
+-- Dynamis - Xarcabard -> Satellite
+INSERT INTO `mob_droplist` VALUES (3151,0,0,1000,3364,20); -- Mystic goad on Satellite Knuckles
```

Comment:
Based on numbers on ffxidb, these goads seem to have a TH0 drop rate between 10-15%, which is higher than the 2% you've got here.

Reply:
I update it  to 10% ;)